app/views/verifyConsentNotificationEmail.scala.html

```diff
@@ -0,0 +1,60 @@
+@(assetsHost: String, assetsBuild: String, studyName: String)
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>@studyName Consent Notification Email Verification</title>
+    <script>
+        window.assetsHost = "@assetsHost";
+    </script>
+    <link href="//@assetsHost/mobile/styles/normalize.css" type="text/css" rel="stylesheet"/>
+    <link href="//@assetsHost/mobile/styles/mobile.css" type="text/css" rel="stylesheet"/>
+</head>
+<body>
+
+<div class="logo_box"><img id="logo" onerror="this.style.visibility='hidden'"/></div>
+<div class="message" id="m1">Verifying...</div>
+<div class="message" id="m2"></div>
```

Comment:
I cleaned up loading the logo so it doesn't happen in JS; we can check this in and I'll make the change, it's trivial.